packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -233,6 +233,12 @@ export interface AnchorNode extends UpPath<AnchorNode>, Listenable<AnchorEvents>
 	 */
 	child(key: FieldKey, index: number): UpPath<AnchorNode>;
 
+	/**
+	 * Gets a child if it exists.
+	 * Does NOT add a ref.
+	 */
+	tryGetChild(key: FieldKey, index: number): AnchorNode | undefined;
```

Comment:
it's a bit odd that child (which isn't named getChild?) returns an UpPath, and tryGetChild returns an AnchorNode. perhaps some naming cleanup could be done here for clarity?

Reply:
I think this makes sense (though I'm biased as I wrote it). child, as a noun named thing, has to be low cost and without side-effects, so it returns the a path to the child, which is the child if there is already an AnchorNode for it. It could be renamed to getChild, but in my opinion thats more verbose, and less informative, since it loses some of the low-cost connotation when adding `get`.

tryGetChild is the 'try' version of that, but try needs a verb form, so we have tryGet, which returns the child AnchorNode if it exists, without forcing one to exist.

We then have getOrCreateChild, which unconditionally returns an AnchorNode.

While I think that all makes since, you are right its a bit inconsistent  due to the "get" being force on the "try" version for mostly grammatical reasons. Perhaps `childIfAnchored` would be a better name for it as it keeps the noun structure? I have done that rename.